src/main.js

```diff
@@ -4,14 +4,82 @@ import { spawn } from 'child_process'
 import parser from 'xml2json'
 import openAboutWindow from 'about-window'
 
-import browsers from './browsers'
+// import browsers from './browsers'
+
+import memFs from 'mem-fs'
+import editor from 'mem-fs-editor'
 
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 let pickerWindow = null
 let tray = null
 let appIsReady = false
 
+let configFileName = 'browserosaurus.json'
+
+var browsers = {}
```

Comment:
Is there a reason `var` has been used instead of `let` and `const`?

Reply:
Not at all, just an habit of mine coming from a long pre-ES6 era.
I think I'm going to do some optimization also on that when I finish my little roadmap.
Thanks for noticing!